core/testing/queryclient/queryclient.go

```diff
@@ -4,40 +4,44 @@ import (
 	"context"
 	"fmt"
 
-	abci "github.com/cometbft/cometbft/api/cometbft/abci/v1"
 	gogogrpc "github.com/cosmos/gogoproto/grpc"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/encoding"
-
-	"github.com/cosmos/cosmos-sdk/client/grpc/reflection"
-	"github.com/cosmos/cosmos-sdk/codec"
-	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 )
 
 var (
 	_ gogogrpc.ClientConn = &QueryHelper{}
 	_ gogogrpc.Server     = &QueryHelper{}
 )
 
+// GRPCQueryHandler defines a function type which handles mocked ABCI Query requests
+// using gRPC
+type GRPCQueryHandler = func(ctx context.Context, req *QueryRequest) (*QueryResponse, error)
+
+// QueryRequest is a light mock of cometbft abci.QueryRequest.
+type QueryRequest struct {
+	Data   []byte
+	Height int64
+}
```

Comment:
would there be any benefits to including the other fields from abci's QueryRequest type?

Reply:
No, this is just mocking the data, and we only use it to marshal the query info these two fields